labs/README.md

```diff
@@ -32,7 +32,8 @@ The above script will do the following:
 Once the script has been executed, you will need to create a new branch, add and commit the new config and then push the changes to GitHub. 
 - Switch to a new branch by using `git switch -c labs-yourGitHubUsername`
 - Run `git status` to check which files have been create and modified.
-- Use `git add ../apps/labs-'yourGitHubUsername'` to stage the newly created files for commit
+- Use `git add ../apps/labs/labs-'yourGitHubUsername'` to stage the newly created files for commit
+- Also commit the changes to kustomization files `git add ../apps/labs/automation/kustomization.yaml ../apps/labs/sbox/base/kustomization.yaml`
```

Comment:
Combine the steps please or just use git add .

Reply:
Updated